plottr/apps/inspectr.py

```diff
@@ -136,7 +141,7 @@ def __init__(self, parent=None):
         self.itemSelectionChanged.connect(self.selectRun)
         self.itemActivated.connect(self.activateRun)
 
-    def addRun(self, runId, **vals):
+    def addRun(self, runId: int, **vals: str) -> None:
```

Comment:
type of `vals` being `str` here is for the "values" of the `**vals` dictionary, right?

Reply:
Yes that is how `**` is typed